WEB/FE/tsconfig.json

```diff
@@ -39,7 +39,7 @@
     // "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
 
     /* Module Resolution Options */
-    // "moduleResolution": "node",            /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
+    "moduleResolution": "node",            /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
```

Comment:
https://www.typescriptlang.org/docs/handbook/module-resolution.html
여기서 찾아보니 default 값이 node라고 하는데 따로 체크해주신 이유가 있을까요?!!

Reply:
```
Classic
This used to be TypeScript’s default resolution strategy.
```
위 링크에 이런 말이 있는걸로 보아 기본값은 classic인 것 아닐까요? 🤔

axios를 사용할 때 타입스크립트가 찾을 수 없어서 에러가 발생했었는데 `moduleResolution`이 classic일 때는 경로가 지정돼있지 않은 파일을 import한 파일의 위치부터 찾아나가기 때문에 axios 패키지를 찾을 수 없고 그 때문에 에러가 발생한 것 같습니다.
 
`moduleResolution`을 node로 설정하면 경로가 지정되어 있지 않은 패키지를 찾을 때 node_modules에서부터 찾기 시작해서 axios를 찾을 수 있는 것 같습니다.

Reply:
다시 확인해보니 
```
You can use the --moduleResolution flag to specify the module resolution strategy. If not specified, the default is Node for --module commonjs, and Classic otherwise
```
commonjs일 때 기본값이 node맞네요! 저부분을 잘 확인못한 것 같네요  ㅎㅎㅎㅎ 👍